rdkafka-sys/build.rs

```diff
@@ -72,6 +72,10 @@ fn main() {
             }
         }
     } else {
+        let rdkafkasys_root = Path::new("rdkafka-sys");
+        if rdkafkasys_root.exists() {
+            assert!(env::set_current_dir(&rdkafkasys_root).is_ok());
+        }
```

Comment:
I don't think this should be here, the build tool should ensure the code is built from the right directory (as cargo does). This probably means a BUILD.bazel file is missing from this directory?